tools/pythonpkg/duckdb/experimental/spark/sql/readwriter.py

```diff
@@ -20,6 +20,47 @@ def saveAsTable(self, table_name: str) -> None:
         relation = self.dataframe.relation
         relation.create(table_name)
 
+    def parquet(self, path: str, mode: Optional[str] = None, partitionBy: Union[str, List[str], None] = None, compression: Optional[str] = None)  -> None:
+        relation = self.dataframe.relation
+        if mode:
+            raise NotImplementedError
+        if partitionBy:
+            raise NotImplementedError
+
+        relation.write_parquet(path, compression=compression)
+    
+    def csv(self, path, mode=None, compression=None, sep=None, quote=None, escape=None,
```

Comment:
Can we please use the exact prototype as PySpark?
<https://spark.apache.org/docs/latest/api/python/reference/pyspark.sql/api/pyspark.sql.DataFrameWriter.csv.html>

This is missing type annotation for the parameters

Reply:
`eea76cf5c174b09d795a4e22e9e791698b8eaaba`